### PR TITLE
0.9.0 master merged into develop

### PR DESCRIPTION
**Pull request by W3stside**
`master` >> `develop` merge

Possible issues:

~1. Not seeing pending orders showing up after sending tx~ **<-- Seeing it now, please confirm**

2. potentially wrong display of orders / lots of merge conflicts there that may been erroneously merged

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr671--dexreact.review.gnosisdev.com

File: src/hooks/useOrders.ts [hunk 1]
```diff
@@ -64,7 +64,8 @@ export function useOrders(): Result {
     const fetchOrders = async (offset: number): Promise<void> => {
       // isLoading is the important one
       // controls ongoing fetching chain
-      if (!userAddress || !networkId || !isLoading) {
+      if (!userAddress || !networkId /*|| !isLoading*/) {
+        // TODO: Do not merge with this, just for doing a quick test!!
```
Comment (alfetopito) [inline src/hooks/useOrders.ts]: Should this have been pushed?

Comment (W3stside) [reply src/hooks/useOrders.ts]: was this @anxolin doing the pending stuff?

Comment (anxolin) [reply src/hooks/useOrders.ts]: Ups, this shouldn't go there

Comment (anxolin) [reply src/hooks/useOrders.ts]: For the merge to develop (this PR), I think it's fine. It'll be eventually merged with the hotfix to develop too

**Pull request merged**